DEVELOPERS.md

```diff
@@ -0,0 +1,32 @@
+# Notes for developers
+
+## System requirements
+
+### `just`
+
+```sh
+# macOS
+brew install just
```

Comment:
yay this worked for me

Reply:
This is the correct `just`. More here: https://github.com/casey/just.